README.md

```diff
@@ -198,9 +198,11 @@ The library comes with the following
 * [Gauge](https://docs.rs/ratatui/latest/ratatui/widgets/struct.Gauge.html)
 * [List](https://docs.rs/ratatui/latest/ratatui/widgets/struct.List.html)
 * [Paragraph](https://docs.rs/ratatui/latest/ratatui/widgets/struct.Paragraph.html)
+* [ScrollBar](https://docs.rs/ratatui/latest/ratatui/widgets/struct.ScrollBar.html)
```

Comment:
This is unrelated to the tree widget. Was this accidentally added or just saw it missing and added it?